ruby-oembed.gemspec

```diff
@@ -27,33 +27,6 @@ Gem::Specification.new do |s|
   s.rubygems_version = "1.8.19"
   s.summary = "oEmbed for Ruby"
 
-  if s.respond_to? :specification_version then
-    s.specification_version = 3
-
-    if Gem::Version.new(Gem::VERSION) >= Gem::Version.new('1.2.0') then
-      s.add_development_dependency(%q<rake>, [">= 0"])
-      s.add_development_dependency(%q<json>, [">= 0"])
-      s.add_development_dependency(%q<xml-simple>, [">= 0"])
-      s.add_development_dependency(%q<nokogiri>, [">= 0"])
-      s.add_development_dependency(%q<rspec>, ["~> 3.0"])
-      s.add_development_dependency(%q<vcr>, ["~> 1.0"])
-      s.add_development_dependency(%q<fakeweb>, [">= 0"])
-    else
-      s.add_dependency(%q<rake>, [">= 0"])
-      s.add_dependency(%q<json>, [">= 0"])
-      s.add_dependency(%q<xml-simple>, [">= 0"])
-      s.add_dependency(%q<nokogiri>, [">= 0"])
-      s.add_dependency(%q<rspec>, ["~> 3.0"])
-      s.add_dependency(%q<vcr>, ["~> 1.0"])
-      s.add_dependency(%q<fakeweb>, [">= 0"])
-    end
-  else
-    s.add_dependency(%q<rake>, [">= 0"])
-    s.add_dependency(%q<json>, [">= 0"])
-    s.add_dependency(%q<xml-simple>, [">= 0"])
-    s.add_dependency(%q<nokogiri>, [">= 0"])
-    s.add_dependency(%q<rspec>, ["~> 3.0"])
-    s.add_dependency(%q<vcr>, ["~> 1.0"])
-    s.add_dependency(%q<fakeweb>, [">= 0"])
-  end
+  s.add_dependency(%q<xml-simple>, [">= 0"])
```

Comment:
**Blocker:** the `xml-simple` gem should be a development dependency, optional for actually installing & using them gem. This suggestion also switches to a use the same syntax as the docs: https://guides.rubygems.org/specification-reference/#add_development_dependency

```suggestion
  s.add_development_dependency 'xml-simple', '>= 0'
```